main.tf

```diff
@@ -288,21 +309,6 @@ resource "spacelift_stack" "default" {
       id        = github_enterprise.value["id"]
     }
   }
-
-  lifecycle {
```

Comment:
You remove this to support the current client's case?

Reply:
Yeah, and it also doesn't make sense to do this check for SingleInstance, so it feels like it's probably more trouble than it's worth? Maybe we should keep it, but let's follow up once client has utilized this new functionality.

Reply:
Agree 👍